ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master__okd.yaml

```diff
@@ -23,6 +27,13 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12
   to: ovn-kubernetes
+- dockerfile_path: Dockerfile.microshift
+  from: ovn-kubernetes-base
+  inputs:
+    ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12
+  to: ovn-kubernetes-singlenode
```

Comment:
dockerfile name is "dockerfile.microshift" but the image produced is "singlenode", seems slightly strange.  @dhellmann do you know the story?

Reply:
@bparees I used singlenode image suffix to indicate the image is optimized for cluster with single node, it could potentially be used in other scenario such as SNO in the future.
The dockerfile.microshift is for developer to easily understand which dockerfile needs to be updated.  I can change the dockerfile name to dockerfile.singlenode if that sounds better?

Reply:
if the image is generically intended to be (potentially) used for "single node topologies" and not just microshift then yes, i'd suggest the dockerfile name reflect that.

but i defer to you and @dhellmann as you have a better understanding of the context here.  @dhellmann can approve the image name

Reply:
@zshi-redhat , do you think we would ever need a different image for MicroShift and SNO OCP?

I think the Dockerfile suffix and image name should match. Calling it "singlenode" is reasonable if we are going to reuse it, and leaves it so we can add a separate "microshift" file later if we have to.

Reply:
> @zshi-redhat , do you think we would ever need a different image for MicroShift and SNO OCP?
> 

Probably yes, I'm not 100% sure about what optimizations shall be applied to SNO at this moment.

> I think the Dockerfile suffix and image name should match. Calling it "singlenode" is reasonable if we are going to reuse it, and leaves it so we can add a separate "microshift" file later if we have to.

I updated the PR to use "-microshift" suffix as image name, will also send request to change the ART build name accordingly.